src/MahApps.Metro/Styles/VS/Expander.xaml

```diff
@@ -313,7 +313,7 @@
         </ObjectAnimationUsingKeyFrames>
     </Storyboard>
 
-    <Storyboard x:Key="MetroExpanderCollapseStoryboard">
+    <Storyboard x:Key="MahApps.Storyboard.MahApps.Styles.Expander.MetroCollapse">
```

Comment:
Shouldn't be this like that? -> `MahApps.Storyboard.Expander.Thumb.MetroCollapse`